.github/autobump.txt

```diff
@@ -232,9 +232,9 @@ bats-core
 batt
 bazarr
 bazel
-bazel@7
 bazel-diff
 bazel-remote
+bazel@7
```

Comment:
This doesn't look intentional

Reply:
oh, this was not sorted before, now it is sorted.

Reply:
actually it got resolved in master, rebased the pr now.